uiowa/uiowa
# [Registrar] Academic calendar dates spanning multiple days.

closes https://github.com/uiowa/uiowa/issues/8042

## To test
1. pull branch and sync registrar
2. got to https://registrar.uiowa.ddev.site/calendars-and-deadlines/academic-calendar
3. observe that all dates in an event that spans multiple days are displayed.
4. Fiddle with things and try to break it!

<details><summary>Gif of the dates being displayed.</summary>


![RegistrarDates](https://github.com/user-attachments/assets/71401f80-8473-4ea1-8ee2-d0dc4da185ac)

</details>
